src/styles/style.css

```diff
@@ -0,0 +1,122 @@
+@import "./star.css";
+
+:root {
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+
+  /* button bg and :hover border+text color */
+  --button-bg-color: #00acdc;
+
+  --button-text-color: #fff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
```

Comment:
reset.css

Reply:
Sure, will create another file for that.